docs/cli/cmdline/script/run.md

```diff
@@ -0,0 +1,51 @@
+---
+title: terramate script run - Command
+description: With the terramate
+
+prev:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+
+next:
+  text: 'Script Tree'
+  link: '/cli/cmdline/script/tree'
+---
+
+# Script Run
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script run SCRIPT-NAME` command will run a Terramate script over a set of stacks. The script will run on all stacks under the current working directory where:
+
+- the script is available (scripts follow the same inheritance rules as globals)
+- any filters match. `script run` currently supports `--changed` and `--tags` filters.
+
+## Usage
+
+`terramate experimental script run [options] NAME`
```

Comment:
```suggestion
`terramate experimental script run [options] NAME...`
```

Reply:
same as above - is this NAME or LABEL? if NAME, we shouldn't use "..." just because we support multi word names

Reply:
that's not the case, see my other comment.

Reply:
resolved